packages/sanity/src/structure/panes/documentList/sheetList/SheetListCell.tsx

```diff
@@ -160,19 +173,6 @@ export function SheetListCell(cell: Cell<DocumentSheetTableRow, unknown>) {
     resetFocusSelection()
   }, [handlePatchField, cellValue, resetFocusSelection])
 
-  // child field inputs can control whether default behavior is stopped or preserved
-  const getOnMouseDownHandler = useCallback(
```

Comment:
Moved this to a custom hook since, a) it requires explicit callout and b) because the mouse handler needs to be added to the Cell itself otherwise the intended behavior isn't as expected since not all inputs span the entire space of a Cell